flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/windowing/TimeWindowTranslationTest.java

```diff
@@ -51,12 +53,48 @@
  */
 public class TimeWindowTranslationTest {
 
+	@Test
```

Comment:
I think this should be covered by the other tests that now exist in this test suite? You probably still have this because these are very recent additions.